book.rb

```diff
@@ -0,0 +1,15 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals << rental
+    rental.book = self
+  end
+end
```

Comment:
- [ ] Great effort working on the project, I would kindly advise you to refactor your code in order to achieve the many-to-many (also has-many-through) relationship. The book itself should be sent as a parameter to create the new rental achieving the 'has-many' association.
a better way of writing that 👇 

```ruby
def add_rental(person, date)
    Rental.new(date, self, person)
  end
```